### PR TITLE
KOGITO-8110 Correct GHA DSL checks

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-8110

- https://github.com/kiegroup/kogito-pipelines/pull/646
- https://github.com/kiegroup/drools/pull/4738
- https://github.com/kiegroup/kogito-runtimes/pull/2547
- https://github.com/kiegroup/kogito-apps/pull/1489
- https://github.com/kiegroup/kogito-examples/pull/1440
- https://github.com/kiegroup/kogito-images/pull/1348
- https://github.com/kiegroup/kogito-operator/pull/1305
- https://github.com/kiegroup/kogito-docs/pull/228
- https://github.com/kiegroup/optaplanner/pull/2242
- https://github.com/kiegroup/optaweb-vehicle-routing/pull/770
- https://github.com/kiegroup/optaplanner-quickstarts/pull/428
- https://github.com/kiegroup/kie-tools/pull/1271

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM, just for this to be changed before merging

File: .ci/jenkins/dsl/test.sh
Comment (rsynek) [inline .ci/jenkins/dsl/test.sh:8]: Does the 'currently' mean it's going to change? Or should it have been 'correctly'?

**Review by rsynek — APPROVED** [review]
One question inline, but does not block merging.

File: .ci/jenkins/dsl/test.sh
Comment (radtriste) [reply .ci/jenkins/dsl/test.sh:8]: oh I think this is a mistake... it should be `correctly`

Comment (radtriste) [reply .ci/jenkins/dsl/test.sh:8]:
```suggestion
  echo "Script must be executed in a Git repository for this script to run correctly"
```

**Pull request merged**
